release-management.md

```diff
@@ -0,0 +1,148 @@
+# go-cose Release Management
+
+## Overview
+
+This document describes [go-cose][go-cose] project release management, which includes release criteria, versioning, supported releases, and supported upgrades.
+
+The go-cose project maintainers strive to provide a stable go-lang implementation for interacting with [COSE][ietf-cose] constructs.
+Stable implies appropriate and measured changes to the library assuring consumers have the necessary functionality to interact with COSE objects.
+If you or your project require added functionality, or bug fixes, please open an issue or create a pull request.
+The project welcomes all contributions from adding functionality, implementing testing to security reviews.
+Please see [here](https://github.com/veraison#contributing) for how to contribute.
+
```

Comment:
Please also state, somewhere in this document, where and how we want the external users to understand what is the current stable `go-cose` release ?

Reply:
is it through the REAMDE.md or somewhere else?

Reply:
I was trying to avoid having to update the readme every time we do a release, and leave it to the [github releases](https://github.com/veraison/go-cose/releases) page. However, the [current readme](https://github.com/veraison/go-cose?tab=readme-ov-file#project-status) does state the current release, so we could continue to update that, or remove it and simply say, check the releases page for the most current.

Do you think we need to update both?

Reply:
I think, it is sensible to add a permanent reference in README.md where to find the latest `go-cose` release!